File: app/src/main/java/vn/com/basc/inco/model/DetailProjectComponent.java
package vn.com.basc.inco.model;

import java.util.List;

/**
 * Created by SONY on 6/13/2016.
 */
public class DetailProjectComponent extends DetailBaseComponent{
    private Detail detail;

    public Detail getDetail() {
        return detail;
    }

    public class Detail{
        String id;
        String type;
        String status;
        List<User> team;

        public String getStatus() {
            return status;
        }

        public String getId() {
            return id;
        }

        public String getType() {
            return type;
        }

        public List<User> getTeam() {
            return team;
        }
    }

}

File: Inco/DetailProjectComponent.swift
//
//  DetailProjectComponent.swift
//  Inco
//
//  Created by admin on 7/31/16.
//  Copyright © 2016 Ban Sac. All rights reserved.
//

import Foundation

class DetailProjectComponent: DetailBaseComponent {
    override init(data:NSDictionary) {
        let extend = data.valueForKey("detail") as? NSDictionary
        self.detail = DetailProjectComponent.Detail(data: extend!)
        super.init(data: data)
    }
    var  detail:DetailProjectComponent.Detail
    class Detail{
        
        var id:String?
        var status:String?
        var type:String?
        var team = [User]()
        init(data:NSDictionary){
            self.id = data.valueForKey("id") as? String
            self.status = data.valueForKey("status") as? String
            self.type = data.valueForKey("type") as? String
            let users = data.valueForKey("team") as? [NSDictionary]
            for user in users! { // loop through data items
                let att = user as NSDictionary
                self.team.append(User(data: att))
            }
            
        }
    }
}